WebDriverAgentLib/Commands/FBSessionCommands.m

```diff
@@ -39,6 +39,10 @@ + (NSArray *)routes
 
     // Health check might modify simulator state so it should only be called in-between testing sessions
     [[FBRoute GET:@"/wda/healthcheck"].withoutSession respondWithTarget:self action:@selector(handleGetHealthCheck:)],
+    
+    //
+    [[FBRoute GET:@"/appium/settings"] respondWithTarget:self action:@selector(handleGetSettings:)],
```

Comment:
we usually add `/wda` prefix to custom endpoints

Reply:
this is part of the appium spec though... i'm open either way